pcsx2/USB/usb-pad/usb-pad-sdl-ff.cpp

```diff
@@ -187,13 +187,13 @@ namespace usb_pad
 				Console.Warning("SDL_HapticUpdateEffect() for constant failed: %s", SDL_GetError());
 		}
 
-		if (!m_constant_effect_running)
-		{
-			if (SDL_HapticRunEffect(m_haptic, m_constant_effect_id, SDL_HAPTIC_INFINITY) == 0)
-				m_constant_effect_running = true;
-			else
-				Console.Error("SDL_HapticRunEffect() for constant failed: %s", SDL_GetError());
-		}
+		// Always 'run' the constant force effect, even when already running. This
+		// mitigates FFB timeout issues experienced by some modern direct-drive
+		// wheels, such as Moza R5, R9, etc...
+		if (SDL_HapticRunEffect(m_haptic, m_constant_effect_id, SDL_HAPTIC_INFINITY) == 0)
+			m_constant_effect_running = true;
+		else
+			Console.Error("SDL_HapticRunEffect() for constant failed: %s", SDL_GetError());
```

Comment:
Should you set m_constant_effect_running to false here now that you try to run the effect while it's still running (true) ?

Reply:
I tried this out for 20 mins on a simagic alpha mini. I believe this has fixed my cutout issues. Thank you very kindly 1000000 times!

Reply:
@F0bes 

Are you referring to whether it should be set to false in the failure case?

In that situation, I don't think we have a hard guarantee of the resulting state, as failing to run an effect may not stop an ongoing one.

Since the flag is primarily used to determine whether to send stop commands to the device, I'd rather err on the side of caution and leave the flag set to 'true' since it's possible the device is still under the influence of an FFB effect.

Reply:
I'm not familiar with this code so I was just ensuring you though of this case. It makes sense not assuming that that failure = not running.

Reply:
Got it, no problem!